samples/snippets/noxfile.py

```diff
@@ -88,7 +85,7 @@ def get_pytest_env_vars() -> Dict[str, str]:
 ALL_VERSIONS = ["2.7", "3.6", "3.7", "3.8", "3.9"]
```

Comment:
Good catch. I filed a request https://github.com/googleapis/synthtool/issues/1118

Reply:
Ah, that noxfile is generated from a template. No, I don't think it's safe to remove Python 2.7 bits, since not all libraries have dropped Python 2 support yet.